### PR TITLE
[SCB-1101] support IPv6 address format

### PR DESCRIPTION
**Pull request by yhs0092**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
Currently our IP address parsing logic cannot work properly in IPv6 environment

### CODE REVIEW

File: foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java [hunk 1]
```diff
@@ -58,6 +60,8 @@
     doGetHostNameAndHostAddress();
   }
 
+  static Pattern ipv6Pattern = Pattern.compile("[\\[][0-9a-fA-F:]*:[0-9a-fA-F:]*:.*[]]");
```
Comment (wujimin) [inline foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java]: why not just let system to check?

File: foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java [hunk 2]
```diff
@@ -58,6 +60,8 @@
...
+
   private static void doGetHostNameAndHostAddress() {
     try {
       doGetIpv4AddressFromNetworkInterface();
```
Comment (wujimin) [inline foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java]: after support ipv6, we still ignore ipv6 addresses?

Comment (yhs0092) [reply foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java]: Should we take ipv6 address into consideration in common situation? Or we may add a switch to let users to decide?

Comment (yhs0092) [reply foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/net/NetUtils.java]: @liubao68  any idea?

**Comment by yhs0092** [comment]
There seems another problem. I need to check the instance ping mechanism.

**Pull request merged**